### ISSUE TITLE
Kicad version is in the future!

### ISSUE THREAD

**Issue by Electro-resonance** [Redherring32/100W-Charging-PCB#1]
Hi, Looks very well designed!

I notice that the KiCAD version is strange:
	(version 20240716)
	(generator "eeschema")
	(generator_version "8.99")
	(uuid "14746920-3a54-4dfc-8a29-611d4cb40dcd")
	
Current released KiCad is 8.05, and 8.99 is rejected by the release versions of kicad.

**Comment by IcyLetters** [comment]
8.99 is a nightly build and can be found here: [Kicad Nightly](https://downloads.kicad.org/kicad/windows/explore/nightlies)

**Comment by Redherring32** [comment]
Yeah it's made with the 8.99 nightly version of Kicad. I probably shouldn't be using nightly for stuff I'm open sourcing, but I love trying out the new
features so darn much. :)
